src/stores/BookmarksStore.ts

```diff
@@ -73,6 +73,7 @@ export class BookmarksStore {
 	@computed
 	public get filteredBookmarks() {
 		const search = this.textSearch.toLowerCase();
+		console.log(this.sortedBookmarks);
```

Comment:
Don't commit commented code

Usually you would also need to bump a patch version in package.json in this case it would be 6.0.14